cmd/kanikoExecute.go

```diff
@@ -101,6 +102,29 @@ func runKanikoExecute(config *kanikoExecuteOptions, telemetryData *telemetry.Cus
 		}
 	}
 
+	log.Entry().Debugf("creating build settings information...")
```

Comment:
there is quite some code duplication.
Do you think it is feasible to extract at least the retrieval of the dockerImage into a dedicated re-use function?

Reply:
Fixed